sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md

```diff
@@ -256,6 +258,27 @@ rename-mapping:
   SubscriptionScopeMetricsRequestBodyParameters: SubscriptionResourceGetMonitorMetricsWithPostContent
   SubscriptionScopeMetric: SubscriptionMonitorMetric
   TimeSeriesElement: MonitorTimeSeriesElement
+  StorageBlobDestination.storageAccountResourceId: -|arm-id
+  StorageTableDestination.storageAccountResourceId: -|arm-id
+  PrivateLinkScopedResource.resourceId: -|arm-id
+  PrivateLinkScopedResource: DataCollectionRulePrivateLinkScopedResourceInfo
+  EventHubDataSource: DataCollectionRuleEventHubDataSource
+  EventHubDestination: DataCollectionRuleEventHubDestination
+  EventHubDestination.eventHubResourceId: -|arm-id
+  EventHubDirectDestination: DataCollectionRuleEventHubDirectDestination
+  EventHubDirectDestination.eventHubResourceId: -|arm-id
+  FailoverConfigurationSpec: DataCollectionRuleBcdrFailoverConfigurationSpec
+  FailoverConfigurationSpec.ActiveLocation: -|azure-location
+  KnownLocationSpecProvisioningStatus: DataCollectionRuleBcdrLocationSpecProvisioningStatus
+  KnownPrometheusForwarderDataSourceStream: DataCollectionRuleKnownPrometheusForwarderDataSourceStream
```

Comment:
Its real name is actually `KnownPrometheusForwarderDataSourceStreams` (with an `s` at the end) therefore this rename is not working
```suggestion
KnownPrometheusForwarderDataSourceStreams: DataCollectionRuleKnownPrometheusForwarderDataSourceStream
```